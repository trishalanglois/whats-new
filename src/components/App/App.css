* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
  color: white;
}

.app {
  background: #000;
  display: grid;
  grid-template-areas:
    'menu search'
    'menu main'
    'footer footer';
  width: 100%;
}

footer {
  border: 1px solid #fff;
  display: block;
  font-style: italic;
  grid-area: footer;
  height: 3em;
  padding: 1em;
  text-align: center;
}
