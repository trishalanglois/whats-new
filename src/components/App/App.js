import React, { Component } from 'react';
import entertainment from '../../data/entertainment';
import health from '../../data/health';
import local from '../../data/local';
import science from '../../data/science';
import technology from '../../data/technology';
import './App.css';
import SearchForm from '../SearchForm/SearchForm';
import Menu from '../Menu/Menu';
import NewsContainer from '../NewsContainer/NewsContainer';

class App extends Component {
  constructor() {
    super();
    this.state = {
      news: local
    }
  }

changeNews = (newsSource) => {
 this.setState({news: newsSource});
}

filterArticles = (event, word) => {
  event.preventDefault();
  const filteredArticles = this.state.news.filter(article => article.description.includes(word) || article.headline.includes(word));
  this.setState({news: filteredArticles})
}

  render () {
    return (
      <main className="app">
        <SearchForm
          filterArticles={this.filterArticles}
        />
        <Menu
          changeNews={this.changeNews}
          local={local}
          technology={technology}
          entertainment={entertainment}
          science={science}
          health={health}/>
        <NewsContainer
          articles = {this.state.news}
        />
        <footer>Icon made by Freepik from www.flaticon.com</footer>
      </main>
    );
  }
}

export default App;
