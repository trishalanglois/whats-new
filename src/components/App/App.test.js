import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { shallow } from 'enzyme';

describe('App', () => {
  it('should create a main section with the correct data', () => {
    const wrapper = shallow( <App />)
    expect(wrapper).toMatchSnapshot();
  })
})
