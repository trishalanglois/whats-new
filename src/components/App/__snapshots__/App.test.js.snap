// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should create a main section with the correct data 1`] = `
<div
  className="app"
>
  <SearchForm
    filterArticles={[Function]}
  />
  <Menu
    changeNews={[Function]}
    entertainment={
      Array [
        Object {
          "description": "The web slinger will be the only superhero with the ability to cross between Disney and Sony’s cinematic universes, according to Marvel Studios.",
          "headline": "Spider-Man Will Remain in the Marvel Cinematic Universe",
          "id": 1,
          "img": "https://static01.nyt.com/images/2019/09/27/multimedia/27xp-spiderman/27xp-spiderman-superJumbo.jpg?quality=90&auto=webp",
          "url": "https://www.nytimes.com/2019/09/27/business/media/spiderman-mcu-sony-marvel.html",
        },
        Object {
          "description": "While the larger conversation around Todd Phillips’ upcoming Joker film continues to intensify as people reflect on how the story’s depiction of a white domestic terrorist fits into our cultural landscape, Warner Bros. is attempting to take back some control of the narrative by barring print and broadcast press access to the red carpet of the Hollywood premiere where interviews were to be conducted.",
          "headline": "Warner Bros. Thinks a ‘Lot Has Been Said’ About Joker, So It’s Canceling Red Carpet Interviews",
          "id": 2,
          "img": "https://i.kinja-img.com/gawker-media/image/upload/s--4KmyUB9z--/c_scale,dpr_2.0,f_auto,fl_progressive,q_80,w_800/sapqo9o5c10gyewaopjg.png",
          "url": "https://io9.gizmodo.com/warner-bros-thinks-a-lot-has-been-said-about-joker-1838543086",
        },
        Object {
          "description": "Snoop’s son Corde Broadus shared the news on Instagram that his baby, Kai Love, died this week just 10 days after his birth.",
          "headline": "Snoop Dogg’s Newborn Grandson Dies Just 10 Days After His Birth",
          "id": 3,
          "img": "https://i.kinja-img.com/gawker-media/image/upload/s--36OH1KfJ--/c_scale,dpr_2.0,f_auto,fl_progressive,q_80,w_800/k5ppabp7phb3cq6vsv49.jpg",
          "url": "https://thegrapevine.theroot.com/snoop-dogg-s-newborn-grandson-dies-just-10-days-after-h-1838547124",
        },
        Object {
          "description": "Metallica have postponed their upcoming tour of Australia and New Zealand so James Hetfield could return to rehab, the heavy metal legends announced Friday.",
          "headline": "Metallica Cancel Tour as James Hetfield Returns to Rehab",
          "id": 4,
          "img": "https://www.rollingstone.com/wp-content/uploads/2019/09/10331207d.jpg?resize=900,600&w=1200",
          "url": "https://www.rollingstone.com/music/music-news/metallica-cancel-tour-james-hetfield-rehab-892158/",
        },
        Object {
          "description": "Deborah Chow will be the director who ushers Ewan McGregor back into the Jedi Universe to resume his portrayal of Obi-Wan Kenobi on the new Lucasfilm series planned for the Disney+ subscription streaming service.",
          "headline": "Obi-Wan: Lucasfilm Taps Jedi Universe Veteran To Direct Disney+ Series",
          "id": 5,
          "img": "https://pmcdeadline2.files.wordpress.com/2019/07/disney-logo.png?w=681&h=383&crop=1",
          "url": "https://deadline.com/2019/09/obi-wan-lucasfilm-taps-jedi-universe-veteran-to-direct-disney-series-1202746939/",
        },
        Object {
          "description": "Like the Pfefferman family, Transparent Musicale Finale is a little too much—okay, a lot too much—but that’s probably warranted, given that this 100-minute installment serves as the curtain call for Jill Soloway’s groundbreaking series.",
          "headline": "The Transparent Musicale Finale is the show at its best, worst, and everything in between",
          "id": 6,
          "img": "https://i.kinja-img.com/gawker-media/image/upload/s--Cg8LXEGx--/c_scale,dpr_2.0,f_auto,fl_progressive,q_80,w_800/crlgajumurx51tsroiz4.jpg",
          "url": "https://tv.avclub.com/transparent-musicale-finale-is-the-show-at-its-best-wo-1838508933",
        },
        Object {
          "description": "Netflix has announced that the sixth season of BoJack Horseman is the final one, and will premiere in two parts. The first arrives on October 25th, while the rest will have to wait until January 31st, 2020. ",
          "headline": "The final season of BoJack Horseman starts October 25th",
          "id": 7,
          "img": "https://o.aolcdn.com/images/dims?quality=85&image_uri=https%3A%2F%2Fo.aolcdn.com%2Fimages%2Fdims%3Fcrop%3D4744%252C3157%252C0%252C0%26quality%3D85%26format%3Djpg%26resize%3D1600%252C1065%26image_uri%3Dhttps%253A%252F%252Fs.yimg.com%252Fos%252Fcreatr-images%252F2019-09%252F35f60b90-e13e-11e9-bef9-1d86bb3b6edb%26client%3Da1acac3e1b3290917d92%26signature%3D4d4debcc8d723f9ac18f6b8eaf605c9cd7de5fbb&client=amp-blogside-v2&signature=17c69548091d526641c527ffcab605bd9d4032ec",
          "url": "https://www.engadget.com/2019/09/27/bojack-horseman-final-season/",
        },
        Object {
          "description": "Prince Harry, Meghan Markle, Prince William, and Kate Middleton appear to have snubbed Princess Beatrice’s engagement on social media.",
          "headline": "Princess Beatrice’s Engagement “Snubbed” by Prince Harry, Meghan Markle, Prince William, and Kate Middleton",
          "id": 8,
          "img": "https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/bea-1569580049.jpeg?crop=0.976xw:0.732xh;0.0102xw,0.0191xh&resize=980:*",
          "url": "https://www.cosmopolitan.com/entertainment/a29260947/princess-beatrice-engagement-snubbed-prince-harry-meghan-markle-prince-william-kate-middleton/",
        },
      ]
    }
    health={
      Array [
        Object {
          "description": "A North Carolina man has died after getting an infection often associated with shellfish, state health officials say.",
          "headline": "Man dies after getting infection linked to shellfish, North Carolina officials say",
          "id": 1,
          "img": "https://www.newsobserver.com/entertainment/restaurants/o6w6n7/picture184859643/alternates/FREE_768/MAIN%20OPTION%202:%20OYSTERS",
          "url": "https://www.bnd.com/news/nation-world/national/article235549897.html",
        },
        Object {
          "description": "U.S. health officials, including Health and Human Services Secretary Alex Azar, are encouraging Americans to get their flu shots early this year.",
          "headline": "Azar, other health officials press people to get flu shots early",
          "id": 2,
          "img": "https://thehill.com/sites/default/files/styles/thumb_small_article/public/flu_012918getty.jpg?itok=1dygJH3l",
          "url": "https://thehill.com/homenews/news/463354-azar-other-public-health-officials-press-people-to-get-flu-shots-early",
        },
        Object {
          "description": "On Friday, Buncombe County Health and Human Services (BCHHS) reports 20 confirmed cases of Legionnaires disease in the county, including one fatality, as of 5 p.m. on Thursday.",
          "headline": "OUTBREAK UPDATE: Confirmed Legionnaires cases in NC, SC increase to more than 60",
          "id": 3,
          "img": "https://wlos.com/resources/media/ba379555-ba08-4d1e-831b-5886e5176db3-large16x9_09.27legionnaires1.jpg?1569600723150",
          "url": "https://wlos.com/news/local/update-legionnaires-cases-in-buncombe-increase-from-15-to-20",
        },
        Object {
          "description": "A human case of West Nile Virus has been confirmed in a Bay County resident.  The Michigan Department of Health and Human Services (MDHHS) confirmed the case. This is the first human case of WNV in Bay County this year, and the fifth this year in Michigan.",
          "headline": "Human case of West Nile Virus reported in Bay County",
          "id": 4,
          "img": "https://bloximages.newyork1.vip.townnews.com/wnem.com/content/tncms/assets/v3/editorial/1/72/172cd3e7-10f3-59a6-8576-94302ee5bedb/5b74e08aea4ad.image.jpg?resize=1700%2C1125",
          "url": "https://www.wnem.com/news/human-case-of-west-nile-virus-reported-in-bay-county/article_4439cea6-e145-11e9-9c58-0b135aae3fb9.html",
        },
        Object {
          "description": "With deer season just around the corner, the Centers for Disease Control are warning hunters about tuberculosis.  Officials with the CDC issued the warning after learning that a man likely contracted tuberculosis after dressing a deer.",
          "headline": "CDC warns hunters after man likely contracted tuberculosis from deer",
          "id": 5,
          "img": "https://tribwxmi.files.wordpress.com/2019/09/gettyimages-621378290.jpg?quality=85&strip=all",
          "url": "https://fox17online.com/2019/09/27/cdc-warns-hunters-after-man-likely-contracted-tuberculosis-from-deer/",
        },
        Object {
          "description": "Babies and young children drank from clay \\"sippy cups\\" during the Bronze Age and the Iron Age, and the practice may have existed as early as 7,000 years ago, a new study reveals.",
          "headline": "Babies Drank from Ancient Sippy Cups Thousands of Years Ago",
          "id": 6,
          "img": "https://cdn.mos.cms.futurecdn.net/5kX2SdvFchtTd34WAmjpHn-1024-80.jpg",
          "url": "https://www.livescience.com/ancient-sippy-cups.html",
        },
        Object {
          "description": "Too much exercising can lead athletes to make bad decisions, according to a new study released in the Journal Current Biology.  The study revealed that endurance athletes who overtrained made more impulsive choices when it came to their finances, eating habits and self-care.",
          "headline": "Too much exercise can lead athletes to make bad decisions: study",
          "id": 7,
          "img": "https://thenypost.files.wordpress.com/2019/09/athletes-510.jpg?quality=90&strip=all&w=1236&h=820&crop=1",
          "url": "https://nypost.com/2019/09/27/too-much-exercise-can-lead-athletes-to-make-bad-decisions-study/",
        },
        Object {
          "description": "Looking on the bright side is more than a tool for taking life’s ups and downs in stride. An optimistic outlook is also good for your health, according to new research.",
          "headline": "Optimism has a surprising benefit: Better heart health, study finds",
          "id": 8,
          "img": "https://s.abcnews.com/images/Health/hospital-patient-gty-jt-190926_hpMain_4x3_992.jpg",
          "url": "https://abcnews.go.com/Health/optimism-surprising-health-benefit-heart-health-study-finds/story?id=65875548",
        },
      ]
    }
    local={
      Array [
        Object {
          "description": "Classic-rock band The Who has postponed its Denver concert, set for Sunday, Sept. 29, after an illness robbed singer Roger Daltrey of his voice, the band said on Twitter.",
          "headline": "The Who postpones Denver Concert at the Pepsi Center",
          "id": 1,
          "img": "https://theknow.denverpost.com/wp-content/uploads/2016/03/CDXXTHEWHODSC_4082x.jpg",
          "url": "https://theknow.denverpost.com/2019/09/27/the-who-postpones-denver-concert/225182/",
        },
        Object {
          "description": "The 24-foot-tall chicken skeleton towers over the people who pass through the main hall of the Denver Central Library. Skeletal wings extend to the side, and it looks downright prehistoric.",
          "headline": "Giant Chicken Skeleton In Denver’s Central Library Stands For More Than Just Dinner",
          "id": 2,
          "img": "https://i0.wp.com/wp-cpr.s3.amazonaws.com/uploads/2019/09/cpr-swolf_black-cube-monumental_DSC8170.jpg?resize=2064,1376",
          "url": "https://www.cpr.org/2019/09/27/giant-chicken-skeleton-in-denvers-central-library-stands-for-more-than-just-dinner/",
        },
        Object {
          "description": "Car2Go, the popular car-sharing and rental company, will be leaving Denver at the end of the October, the company announced Friday.",
          "headline": "Car2Go leaving Denver amid company shake-up",
          "id": 3,
          "img": "https://www.denverpost.com/wp-content/uploads/2017/06/cd20carsharingdsc_8786.jpg?w=859",
          "url": "https://www.denverpost.com/2019/09/27/car2go-leaving-denver/",
        },
        Object {
          "description": "A colorful mural outside Ellis Elementary School pays tribute to a little boy who was known for his smile and kindness to other students.",
          "headline": "Mural outside school pays tribute to Denver boy after tragic death",
          "id": 4,
          "img": "https://ewscripps.brightspotcdn.com/dims4/default/4f3b905/2147483647/strip/true/crop/1280x720+0+0/resize/1280x720!/quality/90/?url=https%3A%2F%2Fewscripps.brightspotcdn.com%2Ff2%2F57%2Fcca3119f42c796342f6a8cbf6358%2Fcaden-mural.jpg",
          "url": "https://www.thedenverchannel.com/news/local-news/mural-outside-school-pays-tribute-to-denver-boy-after-tragic-death",
        },
        Object {
          "description": "A motorcyclist was killed in a crash near East 17th Avenue and Park Avenue in Denver early Friday morning.",
          "headline": "Motorcyclist killed in crash near Capitol Hill in Denver",
          "id": 5,
          "img": "https://localtvkdvr.files.wordpress.com/2019/09/fatal-motorcycle.jpeg?quality=85&strip=all&w=800&h=450&crop=1",
          "url": "https://kdvr.com/2019/09/27/motorcyclist-killed-in-crash-near-capitol-hill-in-denver/",
        },
        Object {
          "description": "Troy Guard, who also owns Guard and Grace, Mister Tuna, Tag and other restaurants, announces a beer and market hall",
          "headline": "The owner of Los Chingones is planning a food hall, beer garden and brewery in Denver Tech Center",
          "id": 6,
          "img": "https://www.denverpost.com/wp-content/uploads/2019/09/CD10TROYGUARD_AC33486x.jpg?w=781",
          "url": "https://www.denverpost.com/2019/09/27/food-hall-denver-tech-center-troy-guard/",
        },
        Object {
          "description": "On Wednesday, a publicist for PokéBar said the Denver stop of the nationally traveling event had been canceled, citing an unspecified directive from Pokémon Nintendo",
          "headline": "Pokémon pop-up bar canceled after organizer cites pushback from Nintendo",
          "id": 7,
          "img": "https://theknow.denverpost.com/wp-content/uploads/2019/09/Pokeman2.jpg",
          "url": "https://theknow.denverpost.com/2019/09/26/pokebar-pokemon-popup-denver-canceled/225040/",
        },
        Object {
          "description": "Medicine Man, a Denver-area dispensary chain with four stores across the state, has halted sales of select vaporizer products, specifically pre-filled concentrate cartridges containing propylene glycol or vitamin E acetate.",
          "headline": "Medicine Man Dispensaries Halt Sales of Select Vape Products",
          "id": 8,
          "img": "https://images1.westword.com/imager/u/745xauto/11494355/mason_jar_winter_collins20180216_049.jpg",
          "url": "https://www.westword.com/marijuana/denver-marijuana-dispensary-pulls-select-vape-products-from-shelves-11494353",
        },
      ]
    }
    science={
      Array [
        Object {
          "description": "The image was captured by NASA's Lunar Reconnaissance Orbiter (LRO) on Sept. 17 as it flew over the targeted landing site of India's Chandrayaan-2 mission.",
          "headline": "India's Lost Moon Lander Is Somewhere in This Photo",
          "id": 1,
          "img": "https://cdn.mos.cms.futurecdn.net/FsUeVGmY89BWeb36U8cUj5-650-80.jpg",
          "url": "https://www.livescience.com/india-moon-landing-site-nasa-photo.html",
        },
        Object {
          "description": "NASA's black hole experts popped up on Reddit on Friday to field questions about these enigmatic objects. ",
          "headline": "NASA, in Reddit AMA, says Earth probably won't be eaten by a black hole",
          "id": 2,
          "img": "https://cnet2.cbsistatic.com/img/tM8V3SxstaN-YQiY1QPVMhYBF1k=/2019/09/26/f41e8d77-b008-4887-ae0b-a6dba3e4db60/bh-visualization1.jpg",
          "url": "https://www.cnet.com/news/nasa-in-reddit-ama-says-earth-probably-wont-be-eaten-by-a-black-hole/",
        },
        Object {
          "description": "A microscopic worm that has three different sexes, manages to survive a dose of arsenic that would kill a person 500 times over, and is able to carry its young inside of its body similar to a kangaroo has been discovered in one of the most uninhabitable places on Earth.",
          "headline": "Worm with three sexes that carries its offspring like a kangaroo discovered",
          "id": 3,
          "img": "https://a57.foxnews.com/static.foxnews.com/foxnews.com/content/uploads/2019/09/1862/1048/worm-with-three-sexes.jpg?ve=1&tl=1",
          "url": "https://www.foxnews.com/science/worm-with-three-sexes-discovered",
        },
        Object {
          "description": "Astronomers have discovered a giant planet that, they say, should not exist, according to current theories.  The Jupiter-like world is unusually large compared with its host star, contradicting a widely held idea about the way planets form.",
          "headline": "Giant planet around tiny star should not exist",
          "id": 4,
          "img": "https://ichef.bbci.co.uk/news/660/cpsprodpb/940B/production/_108999873_planet.jpg",
          "url": "https://www.bbc.com/news/science-environment-49855058",
        },
        Object {
          "description": "Lockheed Martin has won a contract from NASA to produce and operate six Orion spacecraft missions, a program the space agency calls the “backbone” of its plans for deep-space exploration and one that will send the first woman to the moon.",
          "headline": "Lockheed Martin wins NASA contract for Orion program, which will send first woman to the moon",
          "id": 5,
          "img": "https://www.denverpost.com/wp-content/uploads/2018/04/lockheed-04-16-2018-ja-_ja28138a.jpg?w=863",
          "url": "https://www.denverpost.com/2019/09/27/jefferson-county-lockheed-martin-orion-nasa/",
        },
        Object {
          "description": "Three supermassive black holes are poised to collide.  These three monsters stand near one another in a system of merging galaxies that's about a billion light-years from Earth, according to a new study published in The Astrophysical Journal.",
          "headline": "3 monster black holes are going to collide",
          "id": 6,
          "img": "https://a57.foxnews.com/static.foxnews.com/foxnews.com/content/uploads/2019/09/1862/1048/two-supermassive-black-holes-getty-images.jpg?ve=1&tl=1",
          "url": "https://www.foxnews.com/science/3-monster-black-holes-are-going-to-collide",
        },
        Object {
          "description": "A new study—the longest coral reef survey to date—provides an in-depth look at Australia's Great Barrier Reef over the past 91 years.",
          "headline": "Longest coral reef survey to date reveals major changes in Australia's Great Barrier Reef",
          "id": 7,
          "img": "https://scx1.b-cdn.net/csz/news/800/2019/longestcoral.jpg",
          "url": "https://phys.org/news/2019-09-longest-coral-reef-survey-date.html",
        },
        Object {
          "description": "Loss of habitat drives most extinctions. When farms or roads carve forests or fields into fragments, the isolated survivors are more likely to be wiped out. Now, a new study suggests connecting these patches with corridors of suitable habitat can help save populations and species—far more than scientists ever thought.",
          "headline": "Connecting fragmented pieces of habitat can help endangered species recover",
          "id": 8,
          "img": "https://www.sciencemag.org/sites/default/files/styles/inline__699w__no_aspect/public/milkweed_1280p.jpg?itok=5LxrkkYy",
          "url": "https://www.sciencemag.org/news/2019/09/connecting-fragmented-pieces-habitat-can-help-endangered-species-recover",
        },
      ]
    }
    technology={
      Array [
        Object {
          "description": "Any iPhone device from 2011 to 2017 could soon be jailbroken, thanks to an underlying flaw that there's no way to patch.",
          "headline": "Unfixable iOS Device Exploit Is the Latest Apple Security Upheaval",
          "id": 1,
          "img": "https://media.wired.com/photos/5d8e600f34cab0000861c48d/master/w_2560%2Cc_limit/security_jailbreak_MADEPD.jpg",
          "url": "https://www.wired.com/story/ios-exploit-jailbreak-iphone-ipad/",
        },
        Object {
          "description": "SpaceX  has completed joining the bottom and top halves of the Starship orbital-scale prototype it’s building in Boca Chica, Texas.",
          "headline": "SpaceX’s orbital Starship prototype comes together ahead of update event",
          "id": 2,
          "img": "https://techcrunch.com/wp-content/uploads/2019/09/EFfmFEEU8AA1SmA.jpeg?w=1390&crop=1",
          "url": "https://techcrunch.com/2019/09/27/spacexs-orbital-starship-prototype-comes-together-ahead-of-update-event/",
        },
        Object {
          "description": "Google’s Pixel 4 is set to introduce a bunch of new software features, and one of those is a new version of the Pixel Launcher that supports a handy new gesture. Here’s where you can download the launcher from the Pixel 4 for your device.",
          "headline": "Download the updated Pixel Launcher from the Pixel 4",
          "id": 3,
          "img": "https://i2.wp.com/9to5google.com/wp-content/uploads/sites/4/2019/09/pixel_launcher_notfication_gesture_1.jpg?resize=2500%2C0&quality=82&strip=all&ssl=1",
          "url": "https://9to5google.com/2019/09/27/download-updated-pixel-launcher-pixel-4/",
        },
        Object {
          "description": "In its lifetime, the tech giant has gone from a viral startup that quickly became the best way to look up info online to a behemoth corporation that affects so much of daily life",
          "headline": "Google turns 21: A look back at the search engine giant’s iconic life",
          "id": 4,
          "img": "https://thenypost.files.wordpress.com/2019/09/google-birthday.jpg?quality=90&strip=all&w=1236&h=820&crop=1",
          "url": "https://nypost.com/2019/09/27/google-turns-21-a-look-back-at-the-search-engine-giants-iconic-life/",
        },
        Object {
          "description": "While it was only a matter of time, YouTube Music, instead of Google Play Music, is now Android's official music app. Moving forward, Google says YouTube Music will come pre-installed on all new Android smartphones that ship with Android 10 and Android 9 Pie.",
          "headline": "YouTube Music will come pre-installed on all Android 10 phones",
          "id": 5,
          "img": "https://o.aolcdn.com/images/dims?quality=85&image_uri=https%3A%2F%2Fo.aolcdn.com%2Fimages%2Fdims%3Fresize%3D2000%252C2000%252Cshrink%26image_uri%3Dhttps%253A%252F%252Fs.yimg.com%252Fos%252Fcreatr-uploaded-images%252F2019-09%252F55661db0-e136-11e9-8ffe-2d86abf8b6b9%26client%3Da1acac3e1b3290917d92%26signature%3D17b1db00148baef3ce321cc640c231f8c34b233a&client=amp-blogside-v2&signature=f44a8e35982fc23aaeeb23fedfcb7fe928cd4916",
          "url": "https://www.engadget.com/2019/09/27/youtube-music-pre-installed-android-10-phones/",
        },
        Object {
          "description": "If it can do this on a shoestring, imagine what it can do now with real resources.",
          "headline": "Driving the 2020 Lotus Evora GT makes me optimistic about Lotus’ future",
          "id": 6,
          "img": "https://cdn.arstechnica.net/wp-content/uploads/2019/09/Lotus-Evora-1-1440x1080.jpg",
          "url": "https://arstechnica.com/cars/2019/09/driving-the-2020-lotus-evora-gt-makes-me-optimistic-about-lotuss-future/",
        },
        Object {
          "description": "With the start of Apex Legends Season 3 just days away, developer Respawn Entertainment has saved the biggest surprise for last. When the new season starts next Tuesday, players will get to check out the game's brand-new map, World's Edge.",
          "headline": "Apex Legends Season 3 comes with the game's first new map",
          "id": 7,
          "img": "https://o.aolcdn.com/images/dims?quality=85&image_uri=https%3A%2F%2Fo.aolcdn.com%2Fimages%2Fdims%3Fresize%3D2000%252C2000%252Cshrink%26image_uri%3Dhttps%253A%252F%252Fs.yimg.com%252Fos%252Fcreatr-uploaded-images%252F2019-09%252F42299f70-e153-11e9-b3ff-5d8cae067aaa%26client%3Da1acac3e1b3290917d92%26signature%3Da03946bc474c89293d52de16d649b063a7467dd3&client=amp-blogside-v2&signature=94f24c8a021f767555901a9ec265a2e76f60789a",
          "url": "https://www.engadget.com/2019/09/27/apex-legends-season-3-worlds-edge-map/?guccounter=1&guce_referrer=aHR0cHM6Ly9uZXdzLmdvb2dsZS5jb20v&guce_referrer_sig=AQAAAEcGJiwd0FXsz6TI6NorJ9l1NN-UJhnHYDeUtt0m3BBAXoZACJ12FMtiFYMw410n7uvX-EF-PNRNoAKvfVRfMu0GIw58AEQyiB2HEFfKe8pu8oErpE-5QMlt9gNwAXTBQfuzGxGAm3Zc-2sTOtqrCZKapwCIwgGx8O_zHdadPqV6",
        },
        Object {
          "description": "One of Nintendo's mobile strategies is to use iOS and Android as ways to release inferior versions of its console games, something probably best exemplified by Animal Crossing: Pocket Camp",
          "headline": "Two Big Warnings About Mario Kart Tour On iOS And Android",
          "id": 8,
          "img": "https://thumbor.forbes.com/thumbor/960x0/https%3A%2F%2Fblogs-images.forbes.com%2Fdavidthier%2Ffiles%2F2019%2F09%2Fmario-1200x797.jpg",
          "url": "https://www.forbes.com/sites/davidthier/2019/09/27/two-big-warnings-about-mario-kart-tour-on-ios-and-android/#2791019a53ae",
        },
      ]
    }
  />
  <NewsContainer
    articles={
      Array [
        Object {
          "description": "Classic-rock band The Who has postponed its Denver concert, set for Sunday, Sept. 29, after an illness robbed singer Roger Daltrey of his voice, the band said on Twitter.",
          "headline": "The Who postpones Denver Concert at the Pepsi Center",
          "id": 1,
          "img": "https://theknow.denverpost.com/wp-content/uploads/2016/03/CDXXTHEWHODSC_4082x.jpg",
          "url": "https://theknow.denverpost.com/2019/09/27/the-who-postpones-denver-concert/225182/",
        },
        Object {
          "description": "The 24-foot-tall chicken skeleton towers over the people who pass through the main hall of the Denver Central Library. Skeletal wings extend to the side, and it looks downright prehistoric.",
          "headline": "Giant Chicken Skeleton In Denver’s Central Library Stands For More Than Just Dinner",
          "id": 2,
          "img": "https://i0.wp.com/wp-cpr.s3.amazonaws.com/uploads/2019/09/cpr-swolf_black-cube-monumental_DSC8170.jpg?resize=2064,1376",
          "url": "https://www.cpr.org/2019/09/27/giant-chicken-skeleton-in-denvers-central-library-stands-for-more-than-just-dinner/",
        },
        Object {
          "description": "Car2Go, the popular car-sharing and rental company, will be leaving Denver at the end of the October, the company announced Friday.",
          "headline": "Car2Go leaving Denver amid company shake-up",
          "id": 3,
          "img": "https://www.denverpost.com/wp-content/uploads/2017/06/cd20carsharingdsc_8786.jpg?w=859",
          "url": "https://www.denverpost.com/2019/09/27/car2go-leaving-denver/",
        },
        Object {
          "description": "A colorful mural outside Ellis Elementary School pays tribute to a little boy who was known for his smile and kindness to other students.",
          "headline": "Mural outside school pays tribute to Denver boy after tragic death",
          "id": 4,
          "img": "https://ewscripps.brightspotcdn.com/dims4/default/4f3b905/2147483647/strip/true/crop/1280x720+0+0/resize/1280x720!/quality/90/?url=https%3A%2F%2Fewscripps.brightspotcdn.com%2Ff2%2F57%2Fcca3119f42c796342f6a8cbf6358%2Fcaden-mural.jpg",
          "url": "https://www.thedenverchannel.com/news/local-news/mural-outside-school-pays-tribute-to-denver-boy-after-tragic-death",
        },
        Object {
          "description": "A motorcyclist was killed in a crash near East 17th Avenue and Park Avenue in Denver early Friday morning.",
          "headline": "Motorcyclist killed in crash near Capitol Hill in Denver",
          "id": 5,
          "img": "https://localtvkdvr.files.wordpress.com/2019/09/fatal-motorcycle.jpeg?quality=85&strip=all&w=800&h=450&crop=1",
          "url": "https://kdvr.com/2019/09/27/motorcyclist-killed-in-crash-near-capitol-hill-in-denver/",
        },
        Object {
          "description": "Troy Guard, who also owns Guard and Grace, Mister Tuna, Tag and other restaurants, announces a beer and market hall",
          "headline": "The owner of Los Chingones is planning a food hall, beer garden and brewery in Denver Tech Center",
          "id": 6,
          "img": "https://www.denverpost.com/wp-content/uploads/2019/09/CD10TROYGUARD_AC33486x.jpg?w=781",
          "url": "https://www.denverpost.com/2019/09/27/food-hall-denver-tech-center-troy-guard/",
        },
        Object {
          "description": "On Wednesday, a publicist for PokéBar said the Denver stop of the nationally traveling event had been canceled, citing an unspecified directive from Pokémon Nintendo",
          "headline": "Pokémon pop-up bar canceled after organizer cites pushback from Nintendo",
          "id": 7,
          "img": "https://theknow.denverpost.com/wp-content/uploads/2019/09/Pokeman2.jpg",
          "url": "https://theknow.denverpost.com/2019/09/26/pokebar-pokemon-popup-denver-canceled/225040/",
        },
        Object {
          "description": "Medicine Man, a Denver-area dispensary chain with four stores across the state, has halted sales of select vaporizer products, specifically pre-filled concentrate cartridges containing propylene glycol or vitamin E acetate.",
          "headline": "Medicine Man Dispensaries Halt Sales of Select Vape Products",
          "id": 8,
          "img": "https://images1.westword.com/imager/u/745xauto/11494355/mason_jar_winter_collins20180216_049.jpg",
          "url": "https://www.westword.com/marijuana/denver-marijuana-dispensary-pulls-select-vape-products-from-shelves-11494353",
        },
      ]
    }
  />
</div>
`;
